python/tests/test_regs.json

```diff
@@ -37,21 +37,54 @@
                     "-P 50000 --preserve_performance_counters --save_resume "
                 ]
             },
+            "(",
             {
                 "#reg": [
                     "--coin",
-                    "--ftrl",
```

Comment:
[nit] other reductions that may be interesting to add:
--freegrad
--bfgs
--conjugate_gradient